src/main/resources/templates/db_scripts/schema.json

```diff
@@ -313,7 +313,7 @@
     },
     {
       "tableName": "instance",
-      "fromModuleVersion": "17.1.0",
+      "fromModuleVersion": "19.2.0-SNAPSHOT",
```

Comment:
As I understand it, there is no need to change the `fromModuleVersion` of a table when an index is added.

@bohdan-suprun I believe you recently confirmed this?

Reply:
Yes, correct.